pkg/importer/avro.go

```diff
@@ -0,0 +1,49 @@
+package importer
+
+import (
+	"fmt"
+
+	"github.com/anz-bank/sysl/pkg/importer/avro"
+
+	"github.com/arr-ai/arrai/syntax"
+	"github.com/sirupsen/logrus"
+)
+
+// NewAvroImporter returns a new avroImporter
+func NewAvroImporter(logger *logrus.Logger) Importer {
+	return &avroImporter{
+		logger: logger,
+	}
+}
+
+// AvroImporter represents Avro specification importer
+type avroImporter struct {
+	appName string
+	pkg     string
+	logger  *logrus.Logger
+}
+
+// Load returns Sysl result
+func (i *avroImporter) Load(avroSpec string) (string, error) {
+	i.logger.Debugln("Load avro spec")
+
+	val, err := syntax.EvaluateExpr("",
```

Comment:
Building parameters as strings is fragile. We should define a call interface that takes regular Go types and converts them to appropriate arr.ai types and passes them as parameters. It should end up something like:

```
rel.Call(syntax.EvaluateExpr(avro.AvroTransformerScript), avroSpec, i.appName, i.pkg)
```